test/hotspot/jtreg/vmTestbase/nsk/jdi/VirtualMachine/dispose/dispose003a.java

```diff
@@ -135,9 +135,23 @@ public static void main (String argv[]) {
                                  break;
                              } else if (instruction.equals("check_alive")) {
                                  log1("checking on: thread2.isAlive");
-                                 if (test_thread.isAlive()) {
-                                     test_thread.resume();
+                                 // There is no sync between vm.dispose() and test_thread
+                                 // let give thread some time to complete
```

Comment:
nit typo: s/let give/let us give/
Also period at the end of the sentence.

Reply:
comments removed, code moved to util class